src/lib/openjp2/cio.c

```diff
@@ -154,6 +154,13 @@ void opj_read_float_LE(const OPJ_BYTE * p_buffer, OPJ_FLOAT32 * p_value)
         *(l_data_ptr--) = *(p_buffer++);
     }
 }
+/** CPB *******************************************************************/
+/* Added for Omics Data Automation                                        */
+/**************************************************************************/
+
+/**************************************************************************/
+/* End Additions for Omics Data Automation                                */
+/** CPB END ***************************************************************/
```

Comment:
There seem to be no additions here. Is that intended?

Reply:
Originally that did have stuff. Moved all that off to other files or places in source file. Not needed now.